packages/docusaurus-plugin-content-docs/src/sidebars/types.ts

```diff
@@ -45,20 +45,26 @@ type SidebarItemCategoryBase = SidebarItemBase & {
   collapsible: boolean;
 };
 
-export type SidebarItemCategoryLinkDoc = {type: 'doc'; id: string};
+export type SidebarItemCategoryLinkDoc = {
+  type: 'doc';
+  id: string;
+  className?: string;
```

Comment:
Is this new attribute really necessary? What's the advantage of having it instead of reusing the existing category className? Will this be applied to something else that is not the category?